src/app/login/login.component.ts

```diff
@@ -9,28 +11,27 @@ import { AuthProvider } from 'ngx-auth-firebaseui';
 })
 export class LoginComponent implements OnInit {
   providers = AuthProvider;
-  username = '';
-  password = '';
+  userInfo: IUserInfo = {
+    displayName: null,
+    email: null,
+    photoUrl: null,
+    uid: null
+  };
```

Comment:
Is there a need to store it in the component? We're storing it in the service in the `login` function and not using it in the UI from here.